### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by zaxx81**
#### Summary

_**Per MDN JavScript Guidelines**_
There should be no space between a control statement keyword, function, or loop keyword and its opening parenthesis (e.g. `if() { ... }`,` function myFunc() { ... }`, `for(...) { ... })`.

#### Motivation

Reduces conflicting guidance

#### Supporting details

https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/JavaScript

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Josh-Cena** [comment]
That's very interesting because today we have far more usage of `if (...)` than `if(...)` in the docs, and the former is what most formatters prefer as default, and also what [AirBnB](https://github.com/airbnb/javascript#whitespace--around-keywords) recommends. I wonder if the guideline should be updated.

File: files/en-us/web/javascript/reference/statements/if...else/index.md [hunk 1]
```diff
@@ -95,15 +95,15 @@ example:
 
 ```js
 var b = new Boolean(false);
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/reference/statements/if...else/index.md:97]:
```suggestion
const b = new Boolean(false);
```

**Comment by zaxx81** [comment]
> That's very interesting because today we have far more usage of `if (...)` than `if(...)` in the docs, and the former is what most formatters prefer as default, and also what [AirBnB](https://github.com/airbnb/javascript#whitespace--around-keywords), which the guideline cites, recommends. I wonder if the guideline should be updated.

I didn't even think about suggesting the MDN STyle Guide be changed. Google JavaScript Style Guides, W3Schools, and the Standard JS all have a space after `if (...)` also.

**Comment by sideshowbarker** [comment]
> > That's very interesting because today we have far more usage of `if (...)` than `if(...)` in the docs, and the former is what most formatters prefer as default, and also what [AirBnB](https://github.com/airbnb/javascript#whitespace--around-keywords), which the guideline cites, recommends. I wonder if the guideline should be updated.
> 
> I didn't even think about suggesting the MDN Style Guide be changed. Google JavaScript Style Guides, W3Schools, and the Standard JS all have a space after `if (...)` also.

Yeah, the MDN style guide is out of sync with current best practice — and with defaults in Prettier, etc. So what actually needs to change here is the style guide, not the `if...else`.

So the change in this PR branch should not be made, and thus I’m going ahead and closing this.

It would be great if you could instead open a PR for a change to the style guide.

**Comment by zaxx81** [comment]
I agree with the discussion in here. Thank you for the insight, I will make the update to the style guide and submit a pull request there.

**Pull request closed without merging**